src/reducers/versions.tsx

```diff
@@ -721,6 +726,24 @@ type CreateInternalDiffParams = {
   headVersionId: number;
 };
 
+export const createInternalChangeInfo = (
```

Comment:
Nice change but is this needed for this patch?

Reply:
This part of the nested object was not type checked against `DiffInfo` for some reason but when I moved it to get returned from `createInternalChangeInfo()`, it was. It's not necessary for anything in this patch but it was helping me get everything working so it seems worth keeping.